### ISSUE TITLE
FAQ Question: Can the DID Subject of a DID Document be updated/changed after the DID Document has been registered?

### ISSUE THREAD

**Issue by mwherman2000** [w3c/did-core#718]
New FAQ Question for the https://w3c.github.io/did-core/#frequently-asked-questions-about-did-identification section:

Can the DID Subject of a DID Document be updated/changed after the DID Document has been registered?

I presume no?

**Comment by msporny** [comment]
While not explicitly mentioned (this is a DID Method decision)... I have heard of no DID Method that would allow this.

**Comment by OR13** [comment]
example would be DID Subject is the owner of house at this address....

**Comment by OR13** [comment]
certainly the did controller can decide to change what the did identifies. +1 to @talltree comments on the wg call.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-03-30-did#section5-8) on 2021-03-30

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 5.8. FAQ Question: Can the DID Subject of a DID Document be updated/changed after the DID Document has been registered? 

_See github issue [#718](https://github.com/w3c/did-core/issues/718)._




**Brent Zundel:** This would be another thing if drummond can in his refactoring, make sure this question ends up query  
… I think it's clear from the rest of the document... but making it clear would be valuable  

**Drummond Reed:** OK, I will take that one too.  
… Although the answer to earlier one seems obvious, I think there may be different answers for this one, depending on who you ask.  
… Do we have a well-documented reply to answer?  

**Manu Sporny:** This is not explicitly mentioned anywhere I think. It's really DID Method specific.  
… I don't know of any DID method that allows you do change the DID Subject.  
… Could imagine ... tombstoning... to make it seem like that happened.  

**Drummond Reed:** It might be theoretically possible to that, but we don't know of any. Or we could flat out say no.  

**Brent Zundel:** Or you could say whether or not it is possible is DID Method-specific.  

**Kyle Den Hartog:** Advocating for no. Huge can of worms  
… we did a good job avoiding.  

**Brent Zundel:** I know of at least one working group participant who would say yes. If we say DID method-specific, we could avoid that  

**Orie Steele:** I agree with what you just said, brent.  
… Example is DID subject is human owner at house. It changes over time. Does the subject change? I don't think so.  
… But depending on how you read English, you might come up with different ways of answering ti.  

**Markus Sabadello:** I think we have a normative statement somewhere that when you resolve the DID, the id property of the resolved DID document must match what was resolved. If change the DID, feels like this would break. Maybe misunderstanding the question?  

**Drummond Reed:** Tricky question. It feels like two questions: one is is it the same DID document if you change the value of the id property.  
… That's one thing the question could be asking  
… The other, that I thought it was asking, is we said the DID controller controls what the DID identifies. I thought it was asking if that DID controller could make the DID identify something different. That is the one there was a WG participant who would say yes. ... Could be DID method specific  

**Brent Zundel:** I recommend answering as best you can in the PR and then allowing the conversation to continue.  

**Drummond Reed:** I'll make it a separate PR so we can have a discussion just about that.  

</details>

**Comment by mwherman2000** [comment]
> example would be DID Subject is the owner of house at this address: 123 example st... etc...
> 
> The identifier does not change, but the subject does?

The DID Subject of the house at 123 Example Street is not the owner of the house.  It is the house.  The DID (and hence the Subject or DID Subject) of the dwelling at 125 Example Street is the house and will always be that particular house ...even after the house is demolished to create a new convenience store, gas station, fire hall, etc.

POP QUIZ: What is the DID (DID Subject) of the following Driver's License?

![image](https://user-images.githubusercontent.com/6101736/113378492-65c62900-9334-11eb-895b-777e05c44b35.png)

**Comment by mwherman2000** [comment]
The answer is here: https://hyperonomy.com/2021/03/29/tdw-glossary-self-sovereign-identity-ssi-user-scenarios-erin-buys-a-car-in-sovronia-3-user-scenarios/

**Comment by msporny** [comment]
@talltree ping on this issue, can you write a PR to address it?

**Comment by talltree** [comment]
Yes, working on it.

**Comment by msporny** [comment]
PR #755 has been raised to address this issue. This issue will be closed once that PR is merged.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-06-08-did#section5-1) on 2021-06-08

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 5.1. FAQ Question: Can the DID Subject of a DID Document be updated/changed after the DID Document has been registered? 

_See github issue [#718](https://github.com/w3c/did-core/issues/718)._




**Brent Zundel:** I believe there is a PR for this  

**Manu Sporny:** there's a PR, ready to go, but merge conflicts that we're waiting on drummond to fix  

**Drummond Reed:** okay  

</details>

**Comment by msporny** [comment]
PR #755 has been merged. 

@mwherman2000 can you please confirm that the PR raised to address your issue actually addressed your issue?

**Comment by msporny** [comment]
Issue submitter has been non-responsive. WG took action to address the issue by creating and merging a PR that speaks directly to the issue. Closing with implict assumption that the submitter's issue has been addressed.

**Comment by mwherman2000** [comment]
Looks good. Trying to plow through my backlog of 3000 Github notifications :-)

Also been busy writing some code: https://www.youtube.com/watch?v=FFv4WZ0p3aY
